codespell_lib/data/dictionary.txt

```diff
@@ -17735,11 +17735,17 @@ neded->needed
 nedia->media
 nedium->medium
 nediums->mediums
+nedle->needle
+nedles->needles
+nedless->needless
```

Comment:
I'd probably suggest the opposite typo too for each one.

Reply:
Are you suggesting:
```
nedles->needles, needless,
nedless->needless, needles,
```
?

I'm not sure if that makes sense, given that the dictionary (obviously) doesn't also have
```
needles->needless
needless->needles
```
? :shrug:

Reply:
Yes, because when you've typed `nedles` you've already made a mistake. Yes needles is the closest Levenstein distance, but needless is only a little further away and could be likely. If we suggest both, it will prompt and let the user pick, rather than auto-replacing with a possibly incorrect suggestion.

Reply:
Fair enough!